### PR TITLE
商品詳細画面の作成

### PR DESCRIPTION
**Pull request by tkeshun**
管理者用の商品詳細画面を作成。
商品一覧のID部分に詳細へのリンク追加。

### CODE REVIEW

File: src/main/java/com/example/controller/AdminController.java [hunk 1]
```diff
@@ -20,6 +20,7 @@
 import com.example.domain.User;
 import com.example.form.InsertItemForm;
 import com.example.service.AdminService;
+import com.example.service.ShowItemDetailService;
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:23]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'com.example.service.ShowItemDetailService' import. Should be before 'org.springframework.web.bind.annotation.RequestMapping'.

File: src/main/java/com/example/controller/AdminController.java [hunk 2]
```diff
@@ -36,6 +37,9 @@ public class AdminController {
 	@Autowired
 	ShowItemListService itemListService;
 
+	@Autowired
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:40]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:40]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'member def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/com/example/controller/AdminController.java [hunk 3]
```diff
@@ -36,6 +37,9 @@ public class AdminController {
...
+	ShowItemDetailService showItemDetailService;
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:41]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/AdminController.java [hunk 4]
```diff
@@ -56,6 +60,13 @@ public String showItemList(Model model) {
 		return "admin/item_list";
 	}
 
+	@GetMapping("/show-item-detail")
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:63]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:63]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def modifier' has incorrect indentation level 8, expected level should be 2.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:63]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.javadoc.MissingJavadocMethodCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing a Javadoc comment.

File: src/main/java/com/example/controller/AdminController.java [hunk 5]
```diff
@@ -56,6 +60,13 @@ public String showItemList(Model model) {
...
+	public String showItemDetail(Model model, Integer itemId) {
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:64]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/com/example/controller/AdminController.java [hunk 6]
```diff
@@ -56,6 +60,13 @@ public String showItemList(Model model) {
...
+		Item item = showItemDetailService.showItemDetail(itemId);
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:65]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/AdminController.java [hunk 7]
```diff
@@ -56,6 +60,13 @@ public String showItemList(Model model) {
...
+		model.addAttribute("item", item);
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:66]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/AdminController.java [hunk 8]
```diff
@@ -56,6 +60,13 @@ public String showItemList(Model model) {
...
+		return "admin/item_detail";
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:67]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:67]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/com/example/controller/AdminController.java [hunk 9]
```diff
@@ -56,6 +60,13 @@ public String showItemList(Model model) {
...
+	}
```
Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:68]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (tkeshun) [inline src/main/java/com/example/controller/AdminController.java:68]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def rcurly' has incorrect indentation level 8, expected level should be 2.

**Review by uedamiku — APPROVED** [review]

**Pull request merged**